src/printer.ts

```diff
@@ -79,6 +79,8 @@ export interface PugPrinterOptions {
 	readonly pugUseTabs: boolean;
 	readonly bracketSpacing: boolean;
 	readonly pugBracketSpacing: boolean;
+	readonly arrowParens: 'avoid' | 'always';
```

Comment:
Maybe we should extract a `export type ArrowParens = 'avoid' | 'always';` to `src/options/common.ts`

Reply:
Agreed